README.md

```diff
@@ -51,5 +56,9 @@ The service will be accessible at http://localhost:8080/mp-servicemesh-sample/se
     mvn -Pthorntail package 
     docker build -t <docker id>/serviceb-<profile>:<mp version> -f src/main/profiles/thorntail/Dockerfile .
     docker run -p 8080:8080 <docker id>/serviceb-<profile>:<mp version>
+    
+    e.g.
+    docker build -t emijiang/serviceb-thorntail:mp-1.3 -f src/main/profiles/thorntail/Dockerfile .
```

Comment:
Shouldn't that also be 2.0 ?

Reply:
I'm closing this PR and reopened a new PR and address all of you comments @pilhuhn .